projects/plugins/jetpack/extensions/blocks/yoast-promo/index.js

```diff
@@ -0,0 +1,106 @@
+import { PanelBody, PanelRow, ExternalLink, Button } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { PluginPrePublishPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import Gridicon from 'gridicons';
+import { useState } from 'react';
+import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
+import { JetpackYoastLogos } from './JetpackYoastLogos';
+
+const PLUGIN_SLUG_YOAST_FREE = 'wordpress-seo/wp-seo';
+const PLUGIN_SLUG_YOAST_PREMIUM = 'wordpress-seo-premium/wp-seo-premium';
+
+export const YoastPromo = () => {
+	const [ isDismissed, setIsDismissed ] = useState( false );
+	const { isYoastFreeActive, isYoastPremiumActive } = useSelect( select => {
+		const { getPlugin } = select( 'core' );
+
+		const isPluginActive = slug => {
+			const plugin = getPlugin( slug );
+			return plugin && 'active' === plugin.status;
+		};
```

Comment:
Does this work? I haven't tested, but my initial thought was that `getPlugin` was meant to interact with editor plugins like the Jetpack plugin sidebar where this panel is being added, not WordPress plugins.

If that doesn't work, maybe we could rely on https://github.com/Automattic/jetpack/blob/80acbae94f5dd47aabeb7f650313522bc597c53c/projects/packages/forms/src/blocks/contact-form/util/plugin-management.js#L4-L9 ?

Reply:
It should work properly as is. `getPlugin` here is a function exposed by `@wordpress/data` state management library (from `select( 'core' )` functions). Under the hood, it performs an API call to `/wp-json/wp/v2/plugins/<plugin_name>`. Indeed, the function `getPlugin` from `@wordpress/plugins` interacts with editor plugins - and it's confusing they are named similarly.

Reply:
You can also check that function in the console on WordPress admin page.

`wp.data.select('core').getPlugins({per_page: -1})` should return all plugins you have installed in your WP instance with its details.

Reply:
Sweet, TIL!